pkg/lib/ownerutil/util.go

```diff
@@ -406,6 +406,12 @@ func InferGroupVersionKind(obj runtime.Object) error {
 			Version: apiextensionsv1beta1.SchemeGroupVersion.Version,
 			Kind:    "CustomResourceDefinition",
 		})
+	case *operatorsv1.OperatorCondition:
```

Comment:
nit: could we move this case before both CRD versions?

Reply:
Sure!

Reply:
Done in the latest version of the PR.